src/DigitalProductPassport/FoodArtifact/DataSheet_v0.1.py

```diff
@@ -0,0 +1,95 @@
+from definition_tooling.converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class FatContentType(CamelCaseModel):
```

Comment:
Just a suggestion, why not call these classes instead `FatContent` and `EnergyContent`?